### ISSUE TITLE
StackOverflowError on first build

### ISSUE THREAD

**Issue by ibraym** [openremote/custom-project#2]
**Describe the bug**
The build failed with `StackOverflowError` message on fresh copy of the repository 

**To Reproduce**
```
1. git clone git@github.com:openremote/custom-project.git
2. cd custom-project/
3. git submodule init
4. git submodule update
5. chmod u+x gradlew
6. ./gradlew eclipse
```

**Screenshots**
![image](https://user-images.githubusercontent.com/52382283/183233019-c0bd7d43-3e36-49a4-95f1-112e103bccd8.png)

**Setup information:**
 - OS: Ubuntu 22.04
 - java version "17.0.4"
 - OpenRemote v3

**Comment by demenkoeugene** [comment]
I have the same problem

**Comment by richturner** [comment]
This looks like a problem with the eclipse plugin/task. We all use IDEA and haven't used eclipse for some time but can it not just import a gradle project?

**Comment by richturner** [comment]
Should be fixed with [1523b48](https://github.com/openremote/custom-project/commit/1523b48398cf545f1c0a55d38f0e58a34c95b570)
